topics/admin/tutorials/database/slides.html

```diff
@@ -6,6 +6,17 @@
 contributors:
   - martenson
   - nsoranzo
+  - natefoo
+---
+# Galactic Database
+
+.left[Galaxy uses a database for:]
+* Galaxy objects and all their relations (users, histories, datasets, workflows)
+* Job state persistence, job dispatching
+
+.left[Galaxy does not use a database for:]
```

Comment:
```suggestion
.right[Galaxy does not use a database for:]
```

I'm assuming this was the intention?

Reply:
wait, .left != .pull-left. whoops.